.storybook/src/accessibility-tool.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useCallback, useEffect} from 'react'
+import {useGlobals} from '@storybook/api'
+import {IconButton, ThemeProvider, Tooltip} from '../../src'
+import {AccessibilityIcon} from '@primer/octicons-react'
+
+export const ADDON_ID = 'myaddon'
+export const TOOL_ID = `${ADDON_ID}/tool`
+
+export const Tool = () => {
+  const [{showSurroundingElements, colorScheme}, updateGlobals] = useGlobals()
+
+  const toggleMyTool = useCallback(() => {
+    window?.localStorage?.setItem('showSurroundingElements', `${!showSurroundingElements}`)
```

Comment:
Using localStorage to ensure the links are shown on the 'new tab' canvas and to save save users preferences.